README.md

```diff
@@ -12,8 +12,8 @@ Properties
 
 -   **intervals**:
     -   **interval**: After a signal, if another one does not enter the block for this amount of time, emit a timeout signal.
-    -   **repeatable**: If true, a timeout signal is emitted every interval without another input signal instead of just once.
--   **group_by**: Expression proprety. The value by which signals are grouped. Output signals will have *group* set to this value.
+    -   **repeatable**: If `True`, a timeout signal is emitted every interval without another input signal instead of just once.
+-   **group_by**: The value by which signals are grouped. Output signals will have `group` set to this value.
 
 
 Dependencies
```

Comment:
Should the GroupBy Block Mixin dependency still be listed here since we are getting rid of that submodule in 2.0?